### PR TITLE
Adds destruction effects to laces

### PR DESCRIPTION
**Pull request by Jackofashepard**
Splat effects when it gets destroyed

### CODE REVIEW

**Comment by Roland410** [comment]
I personally prefer laces to be more vulnerable. As it stands that was at least one drawback you had. Your lace goes pop, you die. With it having 100 health that's a lot harder to achieve.

**Comment by Sparrowbird611** [comment]
A fair point, but isn't current lace HP only 25? A smartgun matches that number, and could probably pop a lace in two shots.

**Comment by Roland410** [comment]
Yes, but you're more likely to hit the brain, and you'd get a lot of damage reductions anyways, because the skull counts as encasing, which reduces the chance/damage I don't remember by at least half iirc? On top of that the brain has a size of like 60 or something while the lace now has 25.

File: code/modules/organs/internal/stack.dm [hunk 1]
```diff
@@ -58,6 +58,16 @@
...
 
+/obj/item/organ/internal/stack/Destroy()
+	. = ..()
+	var/obj/gore
+	playsound(src, "shatter", 70, 1)
+	gore = new /obj/item/weapon/material/shard(get_turf(owner), MATERIAL_GLASS)
+	gore.throw_at(get_edge_target_turf(src,pick(GLOB.alldirs)),rand(1,3),30)
+	gore = new /obj/effect/decal/cleanable/blood/gibs(get_turf(owner))
+	gore.throw_at(get_edge_target_turf(src,pick(GLOB.alldirs)),rand(1,3),30)
+	owner.visible_message(SPAN_WARNING("[owner]'s neck explodes in a shower of strange blue liquid and metalic fragments!"))
```
Comment (Roland410) [inline code/modules/organs/internal/stack.dm:69]:
```suggestion
	owner.visible_message(SPAN_WARNING("[owner]'s neck explodes in a shower of strange blue liquid and metallic fragments!"))
```

**Comment by BurgerLUA** [comment]
oh lmao I no longer have access

**Pull request merged**